input/resources/measure-exm124-FHIR.json

```diff
@@ -568,7 +568,43 @@
     "use": "official",
     "system": "http://example.org/fhir/cqi/ecqm/Measure/Identifier/cms",
     "value": "124"
-  }, {
+  },
+  {
+    "use":"official",
+    "type": {
+      "coding": [ {
+        "system": "http://hl7.org/fhir/us/cqfmeasures/CodeSystem/identifier-type",
+        "code": "version-independent",
+        "display": "Version Independent"
+      } ]
+    },
+    "system": "urn:ietf:rfc:3986",
+    "value": "http://example.org/fhir/us/cqfmeasures/Measure/EXM124-FHIR"
+  },
+  {
+    "use":"official",
+    "type": {
+      "coding": [ {
+        "system": "http://hl7.org/fhir/us/cqfmeasures/CodeSystem/identifier-type",
+        "code": "version-specific",
+        "display": "Version Specific"
+      } ]
+    },
+    "system": "urn:ietf:rfc:3986",
+    "value": "http://example.org/fhir/us/cqfmeasures/Measure/CMS124-FHIR"
```

Comment:
This should be a version-specific identifier, can we add the version to the end? Otherwise it's a pretty confusing example

Reply:
value has been updated to http://example.org/fhir/us/cqfmeasures/Measure/CMS124v7